libs/react-spa/bff/src/lib/BffProvider.tsx

```diff
@@ -252,6 +244,22 @@ export const BffProvider = ({
       return children
     }
 
+    const error = state?.error
+
+    // Here priority matters. Make sure to check double session screen before error screen.
+    if (
+      showErrorScreen &&
+      error?.statusCode === 409 &&
+      error?.code === LOGIN_ATTEMPT_FAILED_ACTIVE_SESSION
+    ) {
+      return (
+        <BffDoubleSessionModal
+          onSwitchUser={switchUser}
+          onKeepCurrentUser={onRetry}
```

Comment:
hmm 🤔 What do you mean redirect? `onRetry` is already redirecting to the applicationBasePath
```
  const onRetry = () => {
    window.location.href = applicationBasePath
  }
```

Are you meaning just better to clear the error state which would hide the modal? Then I would also need to clear the  error params from the URL. 

I would think reloading the page is simpler and more predictable

Reply:
After going to the code and implementing error reset state, removing error params and replacing history, I realised that in my mind it adds to much overhead to both in state and provider. I would rather like to stick to simple as onRetry.

Reply:
But I updated to the following

```

  const onKeepCurrentUser = useCallback(() => {
    BffError.removeBffErrorParamsFromURL()
    onRetry()
  }, [onRetry])
```

Reply:
I'm sorry, my comment was unclear. I do not like `onRetry`'s current logic of sending the user to the applicationBasePath. We should always send the user to where they were going, when possible. In this case that's fairly easy since that's the current URL.

Eg, signing into `/umsoknir/faedingarorlof/:id`, but due to this screen ending on `/umsoknir`. Users will be quite lost.